app/services/collators/housing_costs_collator.rb

```diff
@@ -1,9 +1,9 @@
 module Collators
   class HousingCostsCollator
     class << self
-      def call(disposable_income_summary:, gross_income_summary:, submission_date:, person:)
+      def call(disposable_income_summary:, gross_income_summary:, submission_date:, person:, allow_negative_net: false)
         housing_calculator = Calculators::HousingCostsCalculator.new(disposable_income_summary:, gross_income_summary:,
-                                                                     submission_date:, person:)
+                                                                     submission_date:, person:, allow_negative_net:)
```

Comment:
As this calculator is only called in one place, the flag could be avoided at this level by moving the [0, net_housing_costs] logic up one level (to this class) and making the decision here. WDYT?

Reply:
Sounds good!